web/src/components/Change.res

```diff
@@ -264,16 +264,70 @@ module RowItem = {
     </tr>
 }
 
+module Pagination = {
+  @react.component @module("@patternfly/react-core")
+  external make: (
+    ~children: 'children=?,
+    ~className: string=?,
+    ~defaultToFullPage: bool=?,
+    ~dropDirection: @string
+    [
+      | @as("up") #Up
+      | @as("down") #Down
+    ]=?,
+    ~firstPage: int=?,
+    ~isCompact: bool=?,
+    ~isDisabled: bool=?,
+    ~isStatic: bool=?,
+    ~isSticky: bool=?,
+    ~itemCount: int,
+    ~itemsEnd: int=?,
+    ~itemsStart: int=?,
+    ~offset: int=?,
+    ~onPerPageSelect: (ReactEvent.Mouse.t, int, int, int, int) => unit=?,
+    ~onSetPage: (ReactEvent.Mouse.t, int, int, int, int) => unit=?,
+    ~onNextClick: (ReactEvent.Mouse.t, int) => unit=?,
+    ~onPreviousClick: (ReactEvent.Mouse.t, int) => unit=?,
+    ~onPageInput: (ReactEvent.Mouse.t, int) => unit=?,
+    ~page: int=?,
+    ~perPage: int=?,
+    ~variant: @string
+    [
+      | @as("top") #Top
+      | @as("bottom") #Bottom
+      | @as("PaginationVariant") #PaginationVariant
+    ]=?,
+    ~widgetId: string=?,
+  ) => React.element = "Pagination"
+}
+
 module Table = {
   @react.component
-  let make = (~store: Store.t, ~changes: list<SearchTypes.change>) =>
-    <table className="pf-c-table pf-m-compact pf-m-grid-md" role="grid">
-      <RowItem.Head />
-      <tbody role="rowgroup">
-        {changes
-        ->Belt.List.mapWithIndex((idx, change) => <RowItem key={string_of_int(idx)} store change />)
-        ->Belt.List.toArray
-        ->React.array}
-      </tbody>
-    </table>
+  let make = (~store: Store.t, ~changes: list<SearchTypes.change>) => {
+    let (page, setPage) = React.useState(_ => 1)
+    let (perPage, setPerPage) = React.useState(_ => 25)
+    let (changeArray, _) = React.useState(_ => Belt.List.toArray(changes))
+    let onSetPage = (_, pageNumber: int, _, _, _) => {
+      setPage(_ => pageNumber)
+    }
+    let onPerPageSelect = (_, perPage: int, _, _, _) => {
+      setPerPage(_ => perPage)
+    }
+    <>
+      <Pagination
```

Comment:
On the board page there can be multiple tables, and hopefully most of them should not be too crowded, so perhaps we should hide the pagination when the list is less than 50 elements? You could use the `React.null` component in that case.

Reply:
Done